### DISCUSSION TITLE
StreamLit

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#23] [General]
Discuss things related to StreamLit here!

**Comment by smrchanda877** [comment]
Hi everyone,

I've created a new branch called "ui-basic". If you're working on any UI-related changes, please push your code to this branch first. This will help us catch and resolve any potential merge conflicts early, before merging into the "main" branch.

Thanks!

**Reply by Jagadeesh-ajjada** [reply]
Sounds good

**Comment by ACraig7** [comment]
StreamLit is [open source](https://docs.streamlit.io/).
